app/views/students/show.html.haml

```diff
@@ -26,6 +26,9 @@
         = "#{@student.team.display_name}"
       - else
         No team
+    %p
+      - if policy(@student).update?
+        = link_to "Edit Profile", edit_student_path(current_user.student.id)
```

Comment:
Here, I would use `edit_student_path(@student)` since that's what you're checking above